blockreassurance.php

```diff
@@ -492,11 +492,11 @@ public function getWidgetVariables($hookName = null, array $configuration = [])
      */
     private function shouldWeDisplayOnBlockProduct($enableCheckout, $enableProduct, $controller)
     {
-        if ($enableProduct === '1' && $controller === 'product') {
+        if ($enableProduct === self::POSITION_BELOW_HEADER && $controller === 'product') {
             return true;
         }
 
-        if ($enableCheckout === '1' && $controller === 'cart') {
+        if ($enableCheckout === self::POSITION_BELOW_HEADER && ($controller === 'cart' || $controller === 'order')) {
```

Comment:
compute both conditions
```suggestion
        if ($enableCheckout === self::POSITION_BELOW_HEADER && in_array($controller, self::ALLOWED_CONTROLLERS)) {
```

Reply:
:ok_hand: